packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -24,10 +24,13 @@ import IntlRelativeFormat from 'intl-relativeformat';
 import { Messages, PlainMessages } from '../messages';
 import { Formats, formats as EN_FORMATS } from './formats';
 import { hasValues, isObject, isString, mergeAll } from './helper';
+import { charMap } from './pseudo_locale';
 
 // Add all locale data to `IntlMessageFormat`.
 import './locales.js';
 
+const CHARS_FOR_PSEUDO_LOCALIZATION_REGEX = /[A-Za-z]|((?<=\[[\s\S]*?\])\([\s\S]*?\))|(<([^"<>]|("[^"]*?"))*?>)/g;
```

Comment:
This is a super complex regex. Could you document what this is supposed to match on (or ignore)? You may also consider breaking this into multiple regexes and concatenating them so you could name each logical part.